### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by OccamsLaser2023** [arkypita/LaserGRBL#2288] [Q&A] [answered]
I'm trying to laser engrave name badges for my company.  I have tried multiple different settings from the recommended acrylic settings of 8/3000/300 to 10/300/200 (and several in between) and they all look fine with ONE exception.  The left edge of the logo looks choppy.  The rest of the logo looks fine and the name and title look fine.  The image I am using is 600dpi gray scale.  I've attached the images.  I just upgraded to the latest version of LaserGRGL (7.8.0).  I'm using and AtomStack A10 Pro.  I've burned through at least six badge blanks and don't want to waste anymore because it is starting to get expensive.  Any help or suggestions would be GREATLY appreciated.

![20240623_150605](https://github.com/arkypita/LaserGRBL/assets/173634410/83e37bc3-9d30-4145-a3cd-4b6d04740268)
![20240623_150556](https://github.com/arkypita/LaserGRBL/assets/173634410/55ac0198-c77e-436a-ab0f-3c69ccdeb0b1)

**Comment by arkypita** [comment]
could you please share your original image file, and some screenshoot of the settings you use?

**Comment by OccamsLaser2023** [comment]
![Larry Siener](https://github.com/arkypita/LaserGRBL/assets/173634410/9fa265f7-d4ba-4200-b027-45000b1f6722)
<img width="587" alt="Screenshot 2024-06-23 154922" src="https://github.com/arkypita/LaserGRBL/assets/173634410/d04da63b-c642-43e1-90fa-ba198711366b">
<img width="219" alt="Screenshot 2024-06-23 155001" src="https://github.com/arkypita/LaserGRBL/assets/173634410/ab8acd60-0b0c-4ea5-b437-db7b1a8b99bc">

I have found that the 300mm/min and the 200 (20%) power work best for the name badges that I engrave.  It used to work fine, but recently it just isn't.

**Reply by arkypita** [reply]
all the parameters you use are correct, the only idea i have is that machine has timing belt too lose, please check

**Comment by OccamsLaser2023** [comment]
Thank you!  Will do!

**Comment by Tribo-2dot0** [comment, marked as answer]
Verifique a correia, como arkypita disse... e para gravações use o Laser Mode M4 (Dynamic Power)

**Reply by OccamsLaser2023** [reply]
It was the Dynamic Power setting!  They are now pristine!  Thank you so much!  You are an AWESOME individual!

**Comment by Bluerdg** [comment]
What program are you using for the design?  I've had similar results when using scans or pictures of a primary design. The image visually looks clean, but digitally the image is distorted due shadows or glare (resulting in variance of gray scale).   Check the redline proof of the image along that edge (after you create image).  If you can identify the same distortions, its a base image issue, if the lines are clean along that edge then it may be a belt or alignment issue on your machine.  If you can burn other projects without similar issue then I'd definitely say it's an image distortion.   I'm assuming you developed a master image that consists of the company logo and name and you're inserting text on that image.

**Comment by OccamsLaser2023** [comment]
Actually, I created all of the images through Powerpoint and save them as
jpgs or png files.  If the need arises, I will do some manipulations in
irfanview (changing the dpi of the image, skewing the image, or converting
to gray scale).  So everything is pretty much digitally created.  I don't
scan images and don't add text to an existing image.  I will definitely be
checking the belt issue as you are the second person to mention it.  I also
think it could be the fixed versus dynamic setting and will be playing
around with that.

Thanks for all of your help!
